notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<p>Follow the standardized way demonstrated in the notebook_template (https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/36455b8125802459f3a40752fbda0e4c9407c854/notebooks/notebook_template.ipynb) to create a Cloud Storage bucket.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Done</p>